libs/openFrameworks/math/ofQuaternion.h

```diff
@@ -199,21 +199,21 @@ ofQuaternion::ofQuaternion() {
 
 
 //----------------------------------------
-ostream& ofQuaternion::operator<<(ostream& os) {
-    os << _v.x << ", " << _v.y << ", " << _v.z << ", " << _v.w;
+ostream& operator<<(ostream& os, const ofQuaternion &q) {
```

Comment:
most likely this line leads to duplicate symbols, since it is not enclosed within a namespace